modules/core/react/lib/InputProvider.tsx

```diff
@@ -1,4 +1,6 @@
 import * as React from 'react';
+// @ts-ignore
```

Comment:
There are no definition files for `element-closest`. We could technically add a `.d.ts` file and exclude it from compilation so we don't ship it. But honestly it's a function that takes a `window`-like object and returns void so I made the tradeoff to ignore it here.

Let me know if you think this should be different.

Reply:
I'm fine with this